src/Stratis.SmartContracts.CLR.Validation/Stratis.SmartContracts.CLR.Validation.csproj

```diff
@@ -13,4 +13,8 @@
     <PackageReference Include="Stratis.SmartContracts.Standards" Version="2.0.0" />
   </ItemGroup>
 
+  <ItemGroup>
+    <ProjectReference Include="..\Stratis.SCL\Stratis.SCL.csproj" />
```

Comment:
Should not have this reference.

Reply:
The validator needs to access types from it.

Reply:
Could it access types by name?